src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -84,45 +151,44 @@
-                                  [:lightbox/navigate-to-lightbox
-                                   token-id
-                                   {:images           [collectible-image]
-                                    :index            0
-                                    :on-options-press #(rf/dispatch [:show-bottom-sheet
-                                                                     {:content
-                                                                      (fn []
-                                                                        [options-drawer/view
-                                                                         {:name  collectible-name
-                                                                          :image preview-uri}])}])}])))}]
-          [header collectible-name collection-name collection-image]
+           {:image-src           preview-uri
+            :container-style     style/preview-container
+            :counter             (utils/collectible-owned-counter total-owned)
+            :native-ID           (when (= animation-shared-element-id token-id) :shared-element)
+            :supported-file?     (utils/supported-file? (:animation-media-type collectible-data))
+            :on-press            (fn []
+                                   (if svg?
+                                     (js/alert "Can't visualize SVG images in lightbox")
+                                     (rf/dispatch
+                                      [:lightbox/navigate-to-lightbox
+                                       token-id
+                                       {:images           [collectible-image]
+                                        :index            0
+                                        :on-options-press #(rf/dispatch [:show-bottom-sheet
+                                                                         {:content
+                                                                          (fn []
+                                                                            [options-drawer/view
+                                                                             {:name collectible-name
+                                                                              :image
+                                                                              preview-uri}])}])}])))
+            :on-collectible-load (fn []
+                                   ;; We need to delay the measurement because the navigation has an
+                                   ;; animation
+                                   (js/setTimeout #(when @title-ref
+                                                     (.measureInWindow @title-ref set-title-bottom))
+                                                  300))}]]
+         [rn/view
+          {:style {:flex             1
+                   :background-color (colors/theme-colors colors/white colors/neutral-95 theme)}}
```

Comment:
Can these styles live in the `style.cljs` file?

Reply:
done